Add-ons/UmbracoForms/Developer/Configuration/index.md

```diff
@@ -4,7 +4,70 @@ With Umbraco Forms it's possible to customize the functionality with various con
 ## Editing configuration values
 The configuration for Umbraco Forms can be changed by modifying the XML based config file found at `/App_Plugins/UmbracoForms/UmbracoForms.config`
 
+### UploadStorageDirectory
+This is *legacy and is no longer in use* - Forms that use an upload field will use the same IFileSystem as the media section - https://our.umbraco.com/documentation/Add-ons/UmbracoForms/Developer/IFileSystem/#forms-containing-upload-fields
```

Comment:
Would be great if it could be as a relative link and also just link on the words like this - 
`as the [media section](../IFileSystem/#forms-containing-upload-fields)`